spinn_front_end_common/interface/interface_functions/compute_energy_used.py

```diff
@@ -72,13 +72,13 @@ def compute_energy_used(machine_allocation_controller=None):
             FecDataView.get_time_scale_factor())
     machine = FecDataView.get_machine()
     db = ProvenanceReader()
-    dsg_time = db.get_category_timer_sum(DATA_GENERATION)
-    execute_time = db.get_category_timer_sum(RUN_LOOP)
+    dsg_time = db.get_category_timer_sum(FecTimer.DATA_GENERATION)
+    execute_time = db.get_category_timer_sum(FecTimer.RUN_LOOP)
     # NOTE: this extraction time is part of the execution time; it does not
     #       refer to the time taken in e.g. pop.get_data() or projection.get()
-    extraction_time = db.get_category_timer_sum(BUFFER)
-    load_time = db.get_category_timer_sum(LOADING)
-    mapping_time = db.get_category_timer_sum(MAPPING)
+    extraction_time = db.get_category_timer_sum(FecTimer.BUFFER)
```

Comment:
There seem to be two categories for this - EXTRACTING and BUFFER.  Are you sure this is the right one?

Reply:
Agreed.

Proves that the timer does not die even on bad info.
Does Log
2022-10-04 14:35:54 ERROR: Incorrect end of category: Extracting when in categoryBufferExtraction

But does need fixing.

Reply:
as above this mismatch is completely gone.
But if it happened again the FecTimer goes boom!